tests/queries/0_stateless/02494_parser_string_binary_literal.sql

```diff
@@ -0,0 +1,30 @@
+select b'';
+select b'0' == '\0';
+select b'00110000'; -- 0
+select b'0011000100110000'; -- 10
+select b'111001101011010110001011111010001010111110010101'; -- 测试
+
+select B'';
+select B'0' == '\0';
+select B'00110000'; -- 0
+select B'0011000100110000'; -- 10
+select B'111001101011010110001011111010001010111110010101'; -- 测试
```

Comment:
You can compare in `sql` and just expect `1` in reference, but current version is also ok
```suggestion
select B'111001101011010110001011111010001010111110010101' == '测试';
```